projects/igniteui-angular/src/lib/splitter/splitter-bar/splitter-bar.component.ts

```diff
@@ -113,6 +202,58 @@ export class IgxSplitBarComponent {
         return !!relatedTabs.find(x => x.resizable === false);
     }
 
+    private moveUpOrLeft() {
+        this.panesInitialization();
```

Comment:
The code (moveUpOrLeft/moveDownOrRight) seems very similar to what is done in the Splitter component onMoveStart/onMoving handlers, which are bound to the moveStart/moving events of the igxSlitterBar.
 Instead of duplicating the same logic is it not possible to emit the events with the related pane this.moveStart.emit(this.pane) and the related delta -  
this.moving.emit(delta) where the delta is +10/-10 depending on the direction?